src/providers/tpm_provider/mod.rs

```diff
@@ -408,6 +410,15 @@ impl Drop for TpmProvider {
     }
 }
 
+lazy_static! {
+    static ref CIPHERS: Vec<Cipher> = {
+        let mut ciphers = Vec::new();
+        ciphers.push(Cipher::aes_256_cfb());
+        ciphers.push(Cipher::aes_128_cfb());
+        ciphers
+    };
+}
+
```

Comment:
I like the clarity of this block but is it worth it to have it instead of doing the operation inside the `find_session_cipher` function? It will be called anyway only each time the provider is built.
I am just being cautious about the number of dependencies that we use 😃

Reply:
Sure, can change! Just wanted one clear place where these are defined so you can clearly see what the values are, but I think I'll actually try and define it as an array and manually define the contents. Clearer, but taking a lot more space